<ui version="4.0" >
 <class>Rda</class>
 <widget class="QMainWindow" name="Rda" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>564</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>RU-District Awardlist</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>31</y>
     <width>491</width>
     <height>507</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item rowspan="2" row="0" column="0" colspan="2" >
     <widget class="QLabel" name="label_2" >
      <property name="font" >
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>  Tabelle f√ºr  "Russian District AWARD"</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2" >
     <widget class="QTableWidget" name="ruAwdTable" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>252</green>
            <blue>244</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>252</green>
            <blue>244</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <column>
       <property name="text" >
        <string>Prefix</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>RDXC</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>  Oblast  </string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>total</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File                       </string>
    </property>
    <addaction name="actionENDE" />
    <addaction name="separator" />
    <addaction name="actionRdAwdListeladen" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>538</y>
     <width>491</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionENDE" >
   <property name="icon" >
    <iconset resource="../qtlogRessourcen.qrc" >
     <normaloff>:/new/prefix1/icon/cancel.png</normaloff>:/new/prefix1/icon/cancel.png</iconset>
   </property>
   <property name="text" >
    <string>ENDE                  </string>
   </property>
  </action>
  <action name="actionRdAwdListeladen" >
   <property name="text" >
    <string>rdAwd-Liste laden</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qtlogRessourcen.qrc" />
 </resources>
 <connections/>
</ui>
